starter-code/index.html

```diff
@@ -1,31 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
+  <div class="main-image">
```

Comment:
Everything down here has to be in another div (it will make your life easy).

It's all about boxes inside boxes inside boxes...